source/mainnet/net/voting/voting.rst

```diff
@@ -61,7 +61,7 @@ These older wallets do not have the capability to connect to dApps. But you do h
 
 .. dropdown:: Desktop wallet
 
-    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Desktop wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before delegating your vote.
+    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Desktop wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before delegating your vote. But the memo transaction which effectively delegates the vote and the vote from the new account need to take place within the official voting window.
```

Comment:
This sentence read a little weird for me. I don't know if theres another form of punctuation more suited for this? 🤷 

```suggestion
    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Desktop wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before delegating your vote. But the memo transaction, which effectively delegates the vote, and the vote from the new account both need to take place within the official voting window.
```